src-static/update_country_info.R

```diff
@@ -11,6 +11,9 @@ box::use(logger[log_info])
 box::use(../src/utils/get_iso3_sf)
 box::use(../src/utils/all_iso3_codes)
 box::use(cs = ../src/utils/cloud_storage)
+box::use(../src/utils/logger)
```

Comment:
How would you implement that specifically? I had tried initially to put these `box::use(logger[log_info])` imports in the `configure_logger` script but it wasn't working.

Reply:
Ah see here I got confused because I misread `box::use(../src/utils/logger)` and was thinking of the package! So was thinking you could not explicitly import `log_info()` but instead call it from the package using `logger$log_info()`.

Reply:
Aha! Yes, good thing we've renamed that now to avoid confusion